charts/add-ons/grafana/templates/grafana.yaml

```diff
@@ -170,7 +170,7 @@ spec:
       - {{- include "partials.proxy" $tree | indent 8 | trimPrefix (repeat 7 " ") }}
       {{ if not .Values.global.cniEnabled -}}
       initContainers:
-      - {{- include "partials.proxy-init" . | indent 8 | trimPrefix (repeat 7 " ") }}
+      - {{- include "partials.proxy-init" $tree | indent 8 | trimPrefix (repeat 7 " ") }}
```

Comment:
`partials.proxy-init` doesn't use any of the values modified in `$tree` anymore, so you can continue passing `. `.

Reply:
That makes sense! Done!